.github/CONTRIBUTING.md

```diff
@@ -112,11 +112,11 @@ To link your Codecademy user profile to GitHub:
 4. Select the Github icon under "Or log in using:"
 5. Proceed through the OAuth process.
 
-## Any tips for a Pull Request (PR)?
+## Any necessary tips for a Pull Request (PR)?
 
 - Before making a PR, make sure you pushed your changes from a branch other than `main`.
 - Name the new branch after the changes being pushed to the PR.
-- Keep your PRs byte-sized. No more than 3 new entries per PR!
+- Keep your PRs byte-sized. No more than 3 new entries per PR! And no more than 3 PRs open at a time!
```

Comment:
There have been cases where contributors were waiting for second reviews and opened more PRs after being assigned to new issues. So I'm thinking "no more than 3 PRs" wouldn't always apply.